mender-convert.sh

```diff
@@ -332,6 +332,11 @@ do_make_sdimg_raspberrypi3() {
   sudo install -d -m 755 ${sdimg_primary_dir}/data
 
   set_fstab $device_type
+
+  [[ $keep -eq "-k" ]] && { rm -rf $output_dir/boot.vfat\
```

Comment:
`-eq` is an arithmetic operator, I don't think it will work with a "-k" argument. Use `=` instead.

Reply:
Done